### ISSUE TITLE
Staging Review finding: At high zoom and narrow viewports, pagination links are cut off

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#97469]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** CMS Team
**Product:** Next Build and Next Build's Events and Event Listing Templates
**Feature:** Events and Event Listing Templates

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the [Outreach Events](https://staging.va.gov/outreach-and-events/events/) page (and, presumably, any other Events pages with enough events listed to have many pages of results), when a user is at high zoom or on a device with a narrow viewport, some pagination links are partially or completely obscured by the edges of the screen. Per WCAG, we want to ensure everything is visible and functional up to 400% zoom / on 320px viewports.

## Recommended action

It looks like the Design System Team is already tracking improving the mobile experience for pagination in a couple of tickets. In the meantime, on the `va-pagination` component I'd recommend setting the `max-page-list-length` prop to no more than 6 (currently 10). From my testing it looks like 6 or smaller displays fine up to 400% zoom.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.4.10 Reflow - Level AA](https://www.w3.org/TR/WCAG21/#reflow)
- **Modality:** Browser zoom
- **Design System Component:** [Pagination ](https://design.va.gov/components/pagination)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
